Silent-Vampires/Welcome-to-Nashville-Silent-Vampires
# Jc restaurant on dom

# Description

populated the restaurant search to the dom, also added the save button with a counter.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

git checkout master
git fetch --all
git checkout jc-restaurant-on-DOM
run http-server
put in a search query into the restaurant search bar
it should populate the dom with 5 results


# Checklist:
